controllers/users.controllers.js

```diff
@@ -26,6 +29,30 @@ const userValidator = Joi.object({
   token: Joi.string(),
 });
 
+const userEmailValidator = Joi.object({
+  email: Joi.string().required(),
+});
+
+const transport = nodemailer.createTransport({
+  host: "sandbox.smtp.mailtrap.io",
+  port: 2525,
+  auth: {
+    user: process.env.USER,
+    pass: process.env.PASS,
+  },
+});
+
+const verificationEmail = async (userEmail, verificationToken) => {
+  const emailOptions = {
+    from: "szymonsliwa@proton.me",
+    to: userEmail,
+    subject: "E-mail verification",
+    html: `Copy the link to verify your email: http://localhost:3000/api/users/verify/${verificationToken}`,
```

Comment:
Tu mogłęś dać to jako link html i http://localhost:3000 powinno być trzymane z .env by potem nie zmieniać w kodzie jak kupisz domenę